pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/TorchToMhlo.cpp

```diff
@@ -796,6 +796,37 @@ LogicalResult ConvertAtenOp<AtenGeluOp>::matchAndRewrite(
   return success();
 }
 
+template <>
+LogicalResult ConvertAtenOp<AtenEmptyMemoryFormatOp>::matchAndRewrite(
+    AtenEmptyMemoryFormatOp op,
+    OpAdaptor adaptor,
+    ConversionPatternRewriter& rewriter) const {
+  Location loc = op.getLoc();
+  SmallVector<int64_t, 4> dshape;
+  if (!matchPattern(op.size(), m_TorchConstantIntList(dshape)))
+    return rewriter.notifyMatchFailure(
+        op, "non-const size parameter unsupported");
+  auto opType = getTypeConverter()
+                    ->convertType(op.getType())
+                    .template cast<RankedTensorType>();
+  if (!opType) {
+    return op.emitError("should be RankedTensorType");
+  }
+  Type resultElementType;
+  if (op.dtype().getType().isa<Torch::NoneType>()) {
```

Comment:
Why should `dtype` force to none?

Reply:
![image](https://user-images.githubusercontent.com/8392447/175448970-0a2e90b9-8181-4c96-a7d0-50f87c990a03.png)

Reply:
only supports None in this PR, will supports more dtype and add an unit test for the empty tensor.